src/board.rb

```diff
@@ -2,7 +2,50 @@
 
 # concrete board class and public interface
 class Board
+  attr_accessor :rows, :board_spaces
+
+  def initialize(rows = 3)
+    @rows = rows
+    @board_spaces = make_example_board
+  end
+
+  def make_board
+    board = {}
+    size = @rows * @rows
+    size.times { |element| board.store(element, format('%02d', element)) }
+    board
+  end
+
+  def make_example_board
+    board = {}
+    size = @rows * @rows
+    size.times { |element| board.store(element, format('%02d', element)) }
+    board
+  end
```

Comment:
These functions are identical – what are the different use cases for them?

Reply:
Make example board os the function that displays the board with the numbers at the beginning of the game.  My intent was to have one function display an example ant the other display a blank board, but I don't think that's necessary any more so I'll take one of them out.